lib/hiera/backend.rb

```diff
@@ -307,8 +307,19 @@ def clear!
 
       def qualified_lookup(segments, hash)
         value = hash
+
         segments.each do |segment|
           throw :no_such_key if value.nil?
+          if segment =~ /^::(.+)/
```

Comment:
Putting
```
options[:scope].keys.each do |key|
  if key =~ /^::(.+)/
    otherkey = $1
  else
    otherkey = "::%s" % key
  end
  options[:scope][otherkey] = options[:scope][key]
end
```
in bin/hiera works. `default_proc` doesn't appear to be called on `include?`.